elated-egrets/bangazon-internal-site-2.0
# added more info to profile page for payment types and order history

## DESCRIPTION
added more user functionality to the profile page

## How to pull down
Commands

```
git fetch --all
git checkout rm-add-links-to-profile
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. make sure you a user logged in and go to the profile page
1. there should now be links for order history, payment type creation and deletion.
1. navigate through the mages and try managing your profile to see if there is anything missing or broken.

## What you changed
added payment types list to the profile page and ability to navigate to the delete and add pages for each. also added links to the order history view. 

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
